src/data/polynomial/algebra_map.lean

```diff
@@ -115,6 +115,15 @@ theorem aeval_def (p : polynomial R) : aeval x p = eval₂ (algebra_map R A) x p
 
 @[simp] lemma aeval_C (r : R) : aeval x (C r) = algebra_map R A r := eval₂_C _ x
 
+lemma aeval_finsum_eq_finsum_aeval {s : finset ℕ} (f : ℕ -> polynomial R) (t : A) :
```

Comment:
This follows directly from `alg_hom.map_sum`. I think you can use that wherever you would have used this lemma.

Reply:
deleted